cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectDockerfile.java

```diff
@@ -264,9 +259,9 @@ private void addOtherArtifact(PrintWriter writer, String connectorPath, OtherArt
         String artifactDir = connectorPath + "/" + artifactHash;
         String fileName = other.getFileName() != null ? other.getFileName() : artifactHash;
         String artifactPath = artifactDir + "/" + fileName;
-        String downloadCmd =  "curl -L --output " + artifactPath + " " + other.getUrl();
+        Cmd downloadCmd =  new Cmd("curl", "-L", "--output").append(artifactPath).append(other.getUrl());
 
-        addUnmodifiedArtifact(writer, other, artifactDir, downloadCmd, artifactPath);
+        addUnmodifiedArtifact(writer, other, artifactDir, downloadCmd.toString(), artifactPath);
```

Comment:
It would be more typesafe to pass `downloadCmd` as a `Cmd` rather than a `String`, that within `addUnmodifiedArtifact` where we use it we know it must already be quoted.